zad1/main.rb

```diff
@@ -1,13 +1,28 @@
 def sort_letters(string)
-
+    return string.split("").sort().join
 end
 
 def count_vowels(string)
-
+    vowels = 'eyuioa'
+    odpowiedz = 0
+    string.split("").each {
```

Comment:
Można by się pozbyć zmiennej `odpowiedz`.Na zwróconym arrayu można  użyć metody [count](https://ruby-doc.org/core-2.5.3/Enumerable.html#method-i-count). Można zrobić to oczywiście jeszcze prościej używając metody `count`, która występuje analogicznie w klasie [String](https://ruby-doc.org/core-2.2.0/String.html#method-i-count)

Reply:
I do wieloliniowych bloków w rubym używa się składni `do end` np.

```ruby
string.split('').each do |letter|
# ...
end